docs/http_api.rst

```diff
@@ -298,6 +298,10 @@ to belong to exactly one organization, but an organization can have many teams.
 
     Get a list of existing organizations
 
+    :queryparam archived:
+        (optional) If true, shows archived organizations. If false, shows
+        organizations that are not archived. Defaults to false.
```

Comment:
Wondering if people would ever want to get a list of all organisations (regardless of whether they are archived or not). One way we could do that would be to show all if `archived` isn't specified, but that would mean each time people want to look at unarchived organisations (probably the most likely use case), they would need to add `archived=true`.

Maybe we could allow multiple values to be given? So `archived=true&archived=false` would show both?

Thoughts?